Sources/Templates/Extensions/MockNamingScheme.swift

```diff
@@ -0,0 +1,26 @@
+public class MockNamingScheme {
+
+    static let shared = MockNamingScheme()
+
+    private var prefix: String = ""
+    private var suffix: String = ""
+
+    private init() { }
+
+    func configure(
+        prefix: String,
+        suffix: String
+    ) {
+        self.prefix = prefix
+        self.suffix = suffix
+
+        if prefix.isEmpty && suffix.isEmpty {
```

Comment:
I think we should accept just anything, if it's nil we default back to what we have now. Like my previous comment